app/js/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default class Button extends React.PureComponent {
```

Comment:
You are completely right about that. There is definitely no need for the custom component. I'm going to replace it with the Button component from react-bootstrap.